### PR TITLE
Modified license

### PR DESCRIPTION
**Pull request by mastrolinux**
License was modified for testing branch creation

### CODE REVIEW

**Comment by akash73** [comment]
license ok

**Comment by OfficineArduino** [comment]
yo

**Comment by AleGueli** [comment]
Approved

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# Test license
                     GNU GENERAL PUBLIC LICENSE
```
Comment (mastrolinux) [inline LICENSE:2]: @akash73 Che ne pensi di questa riga?

Comment (akash73) [inline LICENSE:2]: @mastrolinux perfetta

**Pull request merged**